docs/01-overview/main-areas/eventing/evnt-01-overview.md

```diff
@@ -2,6 +2,35 @@
 title: Overview
 ---
 
-Eventing in Kyma is used to send and receive events from applications. For example, you can subscribe to events from an external application, and when the user performs an action there, you can trigger your Function or microservice. To subscribe to events, you need to use the Kyma [Subscription CR](../../../05-technical-reference/00-custom-resources/evnt-01-subscription.md).
+Eventing, in general, consists of event producers (or publishers) and consumers (or subscribers) that send events to or receive events from an event processing backend. The PubSub messaging pattern, which is the eventing paradigm used in Kyma, does not send messages directly to the subscribers but instead sends them to particular topics (or event types) without any knowledge of existing subscribers. The diagram shows a high level overview of PubSub communication:
 
-Kyma supports [Cloud Events](https://cloudevents.io/) - a common specification for describing event data - and legacy events. Legacy events are converted to Cloud Events by Kyma.
+![PubSub](./assets/pubsub.svg)
+
+The key benefits of PubSub are:
+- communication between loosely coupled components.
+- scale publishers and subscribers easily.
+- asynchronous communication.
```

Comment:
```suggestion
- Asynchronous communication
```
Is this another kind of communication than the one between loosely coupled components? 
Or could those be combined as "Asynchronous communication between loosely coupled components"?

Reply:
Yes its the same. I will combine them